stregsystem/static/stregsystem/bat.js

```diff
@@ -1,38 +1,49 @@
-    d.body.appendChild(z);
-    function R(o,m){return Math.max(Math.min(o+(r()-.5)*400,m-50),50)}
+spawn_bat = function() {
+    var v = document.createElement('video');
+    var s = document.createElement('source');
+    var z = document.createElement('div');
+    var zs = z.style;
+    var a = window.innerWidth * Math.random();
+    var b = window.innerHeight * Math.random();
+    z.classList.add("bat");
+    z.appendChild(v);
+    v.appendChild(s);
+    v.width = "48";
+    v.height = "48";
+    v.autoplay = true;
+    v.loop = true;
+    s.src = media_url + "stregsystem/bat.webm";
+    s.type = "video/webm";
+    document.body.querySelector(".bat-container").appendChild(z);
+
+    function R(o, m) {
+        return Math.max(Math.min(o + (Math.random() - 0.5) * 400, m - 50), 50);
+    }
+
     function A(){
-        var x=R(a,w.innerWidth),y=R(b,w.innerHeight),
-            d=5*Math.sqrt((a-x)*(a-x)+(b-y)*(b-y));
-        zs.opacity=n;n=1;
-        zs.transition=zs.webkitTransition=d/1e3+'s linear';
-        zs.transform=zs.webkitTransform='translate('+x+'px,'+y+'px)';
-        i.style.transform=i.style.webkitTransform=(a>x)?'':'scaleX(-1)';
-        a=x;b=y;
+        var x = R(a, window.innerWidth);
+        var y = R(b, window.innerHeight);
+        var d = Math.round(10 * Math.sqrt((a - x) * (a - x) + (b - y) * (b - y)));
+        zs.opacity = 1;
+        zs.transitionDuration = zs.webkitTransitionDuration = d + 'ms';
+        // zs.transform = zs.webkitTransform = 'translate(' + x + 'px, ' + y + 'px)';
+        zs.left = x + 'px';
+        zs.top = y + 'px';
+        v.style.transform = v.style.webkitTransform = (a > x) ? '' : 'scaleX(-1)';
+        a = x;
+        b = y;
         setTimeout(A,d);
     }
-    setTimeout(A, r() * 3e3);
+    setTimeout(A, Math.random() * 3e3);
 };
 
 d = new Date();
-var NUM_BATS = 2;
+var NUM_BATS = 200;
```

Comment:
We probably shouldn't show 200 bats at once.

Reply:
Probably right xD
It is just test code, I forgot to remove.

Reply:
> We probably shouldn't show 200 bats at once.

At least 400 bats are needed

```suggestion
var NUM_BATS = 400;
```